README.md

```diff
@@ -0,0 +1,59 @@
+📗 Table of Contents
+📖 About the Project
+🛠 Built With
+Tech Stack
+Key Features
+🚀 Live Demo
+💻 Getting Started
+Setup
+Prerequisites
+Install
+Usage
+Run tests
+Deployment
+👥 Authors
+🔭 Future Features
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License
+📖 [HELLO-MICROVERSE]
+Describe your project in 1 or 2 sentences.
+
+[HELLO-MICROVERSE] is a...
+
```

Comment:
- [ ] Kindly 😃 use [the Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. The only optional section is a FAQ. Please remove the template logo on the top, if you have one, you can replace it. This is crucial to have a professional and instructive repository. 👌
- [ ] Also consider after putting it to make sure that the MIT license is working that's for the requirements 💯